index.js

```diff
@@ -1 +1,87 @@
-// implement your API here
+const express = require('express');
+const db = require('./data/db.js');
+
+const server = express();
+
+server.use(express.json());
+
+server.get('/api/users', (req, res) => {
+    db.find()
+    .then(users => {
+        res.status(200).json(users);
+    })
+    .catch(err => {
+        res.status(500).json({ message: 'The users information could not be retrieved.'});
+    })
+})
+
+server.get('/api/users/:id', (req, res) => {
+    const { id } = req.params;
+    db.findById(id)
+    .then(user => {
+        res.status(200).json(user);
+    })
+    .catch(err => {
+        res.status(404).json({ message: 'The user with the specified ID does not exist.'})
```

Comment:
careful here, there are THREE cases you need to account for in this endpoint, not just two: server error belongs in the .catch (status 500, internal error), successful request AND user exists sends status 200 along with the user, while successful request and user _doesn't_ exist (status 404) also belongs in the .then block